### PR TITLE
Import addLibraryVersionToPerseusDebug from local file, not package

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

While implementing our production version diagnostics for Perseus, I used some shell scripts to generate out the code for each package. In doing so, I implemented the `version.ts` for `perseus-core` incorrectly (as that's the package that hosts the helper function). 

All tests pass locally but when Perseus is integrated into a host application, it blows up with the error: `TypeError: perseusCore.addLibraryVersionToPerseusDebug is not a function` (@nedredmond).

Issue: "none"

## Test plan:

`yarn test` 
Let the snapshot build and then try it in webapp.

### CODE REVIEW

File: packages/perseus-core/src/__tests__/version.test.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    it("adds the perseus-core version to globalThis", async () => {
+        await import("../version");
+
+        expect(globalThis).toHaveProperty("__perseus_debug__");
+        expect(globalThis.__perseus_debug__).toMatchInlineSnapshot(`
+            {
+              "@khanacademy/perseus-core": "__lib_version__",
+            }
+        `);
+    });
```
Comment (jeremywiebe) [inline packages/perseus-core/src/__tests__/version.test.ts:15]: I'm not 100% sure if this is helpful.  If I revert the change in `version.ts` it still passes so this doesn't actually protect against regression here.  

Any thoughts?

Comment (benchristel) [reply packages/perseus-core/src/__tests__/version.test.ts:15]: Modifying `globalThis` in a test makes me nervous. The fact that the test doesn't catch regressions makes me inclined to remove it.

I'd rather lean on a linter or something to check invariants like "a package should never import its own code from `node_modules`" (if that is in fact what we want to enforce... I'm not sure what the rule should be).

**Review by benchristel — APPROVED** [review]
Looks good! I'd lean towards removing the test.

File: packages/perseus-core/src/version.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // version number during the release build.
 // In dev, you'll never see the version number.
 
-import {addLibraryVersionToPerseusDebug} from "@khanacademy/perseus-core";
+import {addLibraryVersionToPerseusDebug} from "./utils/add-library-version-to-perseus-debug";
```
Comment (nedredmond) [inline packages/perseus-core/src/version.ts:5]: Oh, I see the problem now!

Comment (nedredmond) [reply packages/perseus-core/src/version.ts:5]: Thanks for the quick fix!

**Review by nedredmond — APPROVED** [review]

File: packages/perseus-core/src/__tests__/version.test.ts (hunk 1 above)
Comment (jeremywiebe) [reply packages/perseus-core/src/__tests__/version.test.ts:15]: You've convinced me. I'll remove the test and land this.  I'll also do a small amount of research if this would be relatively easy to lint for (maybe with ESLint's dynamic rules).
